### PR TITLE
Update sqlalchemy stubs

### PR DESCRIPTION
**Pull request by WouldYouKindly**
I believe this covers both [647](https://github.com/python/typeshed/pull/647) and [719](https://github.com/python/typeshed/pull/719). 

Moved sqlalchemy stubs from 2 to 2and3; 
changed sqlalchemy/util/compat.pyi:37 to be of type typing.Text;
added incomplete stubs for sqlalchemy.orm.utils and slqalchemy.sql.elements.ColumnElement.

### CODE REVIEW

**Comment by gvanrossum** [comment]
Thanks!!! I've verified this only to the extent that it doesn't cause regressions for our internal codebases.

 I'll now close #719.

Regarding #647, do you feel up to extracting the pymysql stubs from that and submitting those as a separate PR?

**Comment by WouldYouKindly** [comment]
@gvanrossum Yes. I'm hoping to send a PR before/on the weekend.

File: third_party/2and3/sqlalchemy/sql/elements.pyi [hunk 1]
```diff
@@ -7,7 +9,38 @@ from .. import util
...
 class ClauseElement(Visitable): ...
 
-class ColumnElement(ColumnOperators, ClauseElement): ...
+class ColumnElement(ColumnOperators, ClauseElement):
+    __visit_name__ = 'column'  # type: Text
+    primary_key = False  # type: Any
+    foreign_keys = []  # type: Any
+    _label = None
+    _key_label = key = None
+    _alt_names = ()
```
Comment (ambv) [inline third_party/2and3/sqlalchemy/sql/elements.pyi]: Missed variable annotations above are failing mypy's own tests.

Comment (WouldYouKindly) [inline third_party/2and3/sqlalchemy/sql/elements.pyi]: Guess I'll just annotate them with Any for now... I'll submit a PR in a few days.

Comment (ambv) [inline third_party/2and3/sqlalchemy/sql/elements.pyi]: I already did, no worries! Just keep this in mind for future PRs.

**Pull request merged**